lyncs_io/dask_io.py

```diff
@@ -58,9 +55,8 @@ def save(self, array, chunks=None):
                 self.filename,
                 array,
                 offset=offset,
-                sl=sl[block],
+                sl=slices[block],
             )
-
             tasks.append(task)
 
         return tasks
```

Comment:
I did try to create a dictionary with keys and tasks as dask does it internally but eventually it resulted in the same `tasks` list so I 'm keeping that one. I don't know though how you can visualize a graph if possible. Neither of the two seemed to produced something as good looking as the ones from dask routines.

Reply:
Ok, I see. We should check this together.